lte/gateway/c/oai/tasks/nas/emm/sap/emm_cn.c

```diff
@@ -427,17 +430,13 @@ static int _emm_cn_pdn_config_res(emm_cn_pdn_config_res_t *msg_pP)
       OAILOG_FUNC_RETURN(LOG_NAS_EMM, rc);
     }
     if (!is_pdn_connectivity) {
-      nas_itti_pdn_connectivity_req(
-        emm_ctx->esm_ctx.esm_proc_data->pti,
-        msg_pP->ue_id,
-        pdn_cid,
-        &emm_ctx->_imsi,
-        emm_ctx->_imeisv,
-        emm_ctx->esm_ctx.esm_proc_data,
-        emm_ctx->esm_ctx.esm_proc_data->request_type);
+      if ((mme_app_send_s11_create_session_req(
+        mme_app_desc_p, ue_mm_context, pdn_cid)) == RETURNok) {
+        increment_counter("mme_spgw_create_session_req", 1, NO_LABELS);
+      }
     } else {
+      OAILOG_ERROR(LOG_NAS_ESM, "Received Invalid PDN type \n");
```

Comment:
Is the error here that PDN type is invalid or that a matching PDN context could not be found?

Reply:
Changed the variable name to "is_pdn_context_exists_for_apn" to be more significance to functionality

The else part is not required. Because in case if pdn_context doesn't exist for apn, we are creating pdn context at line number 380 (in updated file) and sending create session request to SPGW. 
It's nothing related to either PDN type and PDN context id